tests/conftest.py

```diff
@@ -0,0 +1,82 @@
+import os
+import sys
+
+import mock
+import click
+import pytest
+
+import mock_tables.dbconnector
+
+import sonic_device_util
+from swsssdk import ConfigDBConnector
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+sys.path.insert(0, modules_path)
+
+generated_services_list = [
+    'ntp-config.service',
+    'warmboot-finalizer.service',
+    'watchdog-control.service',
+    'rsyslog-config.service',
+    'interfaces-config.service',
+    'hostcfgd.service',
+    'hostname-config.service',
+    'topology.service',
+    'updategraph.service',
+    'config-setup.service',
+    'caclmgrd.service',
+    'procdockerstatsd.service',
+    'pcie-check.service',
+    'process-reboot-cause.service',
+    'dhcp_relay.service',
+    'snmp.service',
+    'sflow.service',
+    'bgp.service',
+    'telemetry.service',
+    'swss.service',
+    'database.service',
+    'database.service',
+    'lldp.service',
+    'lldp.service',
+    'pmon.service',
+    'radv.service',
+    'mgmt-framework.service',
+    'nat.service',
+    'teamd.service',
+    'syncd.service',
+    'snmp.timer',
+    'telemetry.timer']
+
+
+def run_command(command, display_cmd=False, return_cmd=False):
+    if display_cmd == True:
+        click.echo(click.style("Running command: ", fg='cyan') + click.style(command, fg='green'))
```

Comment:
Is it possible to create a fixture to do this?
Also, can you use `mock` to fake the execution of the command instead of redefining it?

Reply:
mock is trying to mock the input, I cannot find the mock support mock function.

Reply:
Your new method of reassigning the function is more clear, and I'd be OK with it. But I believe you should be able to use `mock.patch` like this: https://github.com/Azure/sonic-utilities/pull/1021/files#diff-ec4cac544e7af7e612e78e8480cd701dR48

Reply:
the mock is used to get specific return value from the mocked function.

Reply:
It doesn't actually call the function, it just provides the return value as if the function returned it. I think you should just be able to set `return_value=None` and the function will basically become a no-op which returns None.

But I'm OK with your current approach, also.

Also, just a note, the `mock` library is now a part of `unittest` as of Python 3.3. So if we continue using `mock`, we will need to import `unittest` anyway.

Reply:
mock.patch is still used for mock the return value. In this case, we do not need to mock the return value.

Reply:
True. I just suggested making the function a no-op using `mock.patch` also. But your method is clear, so it's fine.

Reply:
no-op won't work for this case, since we still want to the function to output the command so that we can compare with the expected output.

Reply:
True. For the commands which set --verbose by default. But that's only a few commands. If only there was an easy way to capture the acutal commands being issued, rather than the stdout...